src/c4/C4_Functions.hh

```diff
@@ -466,6 +466,18 @@ DLL_PUBLIC_c4 bool isScalar();
 //! Return the processor name for each rank.
 DLL_PUBLIC_c4 std::string get_processor_name();
 
+//---------------------------------------------------------------------------//
+// prefix_sum
+//---------------------------------------------------------------------------//
+/*!
+ * \brief Return the value of the prefix sum at this processor.
+ *
+ * \param node_value
+ * Current node's value of variable to be prefix summed
```

Comment:
Put the description on the same line as the `\param` or check that this produces the correct output (make autodoc).  See examples at https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html.